ms-common-auth/src/main/java/net/trajano/ms/authz/Authorization.java

```diff
@@ -0,0 +1,28 @@
+package net.trajano.ms.authz;
+
+import net.trajano.ms.Microservice;
+import net.trajano.ms.authz.internal.CacheNames;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.trajano.ms.authz.internal.CacheNames'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)